backend/test/unit/controller/levelController.test.ts

```diff
@@ -0,0 +1,92 @@
+import { expect, test, jest, afterEach } from '@jest/globals';
+import { Response } from 'express';
+
+import { handleLoadLevel } from '@src/controller/levelController';
+import { LevelGetRequest } from '@src/models/api/LevelGetRequest';
+import { ChatModel } from '@src/models/chat';
+import { ChatMessage } from '@src/models/chatMessage';
+import { Defence } from '@src/models/defence';
+import { EmailInfo } from '@src/models/email';
+import { LEVEL_NAMES } from '@src/models/level';
+
+declare module 'express-session' {
+	interface Session {
+		initialised: boolean;
+		chatModel: ChatModel;
+		levelState: LevelState[];
+	}
+	interface LevelState {
+		level: LEVEL_NAMES;
+		chatHistory: ChatMessage[];
+		defences: Defence[];
+		sentEmails: EmailInfo[];
+	}
+}
```

Comment:
Hmm, interesting. This is now replicated in implementation and test code, so maybe we need to be extracting the typedef into a shared location that both tsconfig can refer to. Feel free to tackle that later though.